Snippets/ASQ/ASQ_7/Configuration/MessageWrapperSerializer.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace ASQ_7.Configuration
+{
+    using NServiceBus;
+
+    public class MessageWrapperSerializer
+    {
+        #region SerializerAndMessageWrapperSerializer
```

Comment:
Do you need to wrap the whole function as a snippet? Or can you just show just the code snippet for the config?